### PR TITLE
Fix guides from List_of_Guides folder

### PR DESCRIPTION
**Pull request by TPGPL**
yeah, some things are outdated

### CODE REVIEW

File: wiki/List_of_Guides/Searching_and_Downloading_Beatmaps/zh.md [hunk 1]
```diff
@@ -43,53 +45,65 @@ Osu!谱面搜索引擎对于关键词采用“或”的关系进行搜索，即
 
 所谓“尊重原作”，即输入歌曲最原本的相关信息，而不是使用译名/非官方名。
 
-`例1：`
-` `**`√`**` 初音ミク`
-` `**`×`**` 初音未来`
+```
+例1：
+ **√** 初音ミク
```
Comment (kj415j45) [inline wiki/List_of_Guides/Searching_and_Downloading_Beatmaps/zh.md]: image

File: wiki/List_of_Guides/Searching_and_Downloading_Beatmaps/zh.md [hunk 2]
```diff
@@ -43,53 +45,65 @@ Osu!谱面搜索引擎对于关键词采用“或”的关系进行搜索，即
...
 如果官方发布专辑时有相关网站，可以访问网站查看专辑内各歌曲标题等信息。
 
 此外还有一些专辑百科类的网站，例如针对动漫/游戏音乐的 <http://vgmdb.net/> 等。
 
-**`例`**` 搜索专辑歌曲《My Love》（月度作图比赛歌曲）`
-`搜索歌手和歌名可以转到专辑信息页面：http://kubaoms.bandcamp.com/album/adhd`
-`可以知道以下信息：歌手Kuba Oms，歌名My Love，专辑ADHD`
-`因此搜索关键词应该填写：Kuba Oms My Love ADHD`
+```
+**例** 搜索专辑歌曲《My Love》（月度作图比赛歌曲）
```
Comment (kj415j45) [inline wiki/List_of_Guides/Searching_and_Downloading_Beatmaps/zh.md]: the same like ↑↑↑

File: wiki/List_of_Guides/Searching_and_Downloading_Beatmaps/zh.md (hunk 1 above)
Comment (TPGPL) [inline wiki/List_of_Guides/Searching_and_Downloading_Beatmaps/zh.md]: uh

Comment (TPGPL) [inline wiki/List_of_Guides/Searching_and_Downloading_Beatmaps/zh.md]: so probably i'll delete code blocks

Comment (RaikireHiuduo) [inline wiki/List_of_Guides/Searching_and_Downloading_Beatmaps/zh.md]: Most likely a false alarm: https://osu.ppy.sh/wiki/ZH:Searching_and_Downloading_Beatmaps

Probably would want to delete all GFM styling in the code blocks.

Comment (TPGPL) [inline wiki/List_of_Guides/Searching_and_Downloading_Beatmaps/zh.md]: you mean the bold syntax?

Comment (RaikireHiuduo) [inline wiki/List_of_Guides/Searching_and_Downloading_Beatmaps/zh.md]: yes.

**Pull request merged**